### PR TITLE
Epic 7 - Yana Vyklyuk

### PR DESCRIPTION
**Pull request by yanavyklyuk**

### CODE REVIEW

**Comment by okohuch** [comment]
стоп

**Comment by izagor** [comment]
+ 12-19-2023

File: ai_12/yana_vykliuk/epic_7_yana_vyklyuk/kr.cpp [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+#include <ctime>
+#include <fstream>
+
+const int cols = 10;
+const int rows = 10;
+const char filename[] = "file.txt";
+
+using namespace std;
+
+int** create_matrix(const int number)
```
Comment (fm2lpnu) [inline ai_12/yana_vykliuk/epic_7_yana_vyklyuk/kr.cpp:13]: Task 1 (a,b) is not implemented according to requirements.
Int is implemented, but float implementation is missing

File: ai_12/yana_vykliuk/epic_7_yana_vyklyuk/kr.cpp [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                fi = 0; // or some other suitable value
+            }
+            matrix[i][j] = fi;
+        }
+    }
+
+    return matrix;
+}
+
+void sorting(int** matrix)
```
Comment (fm2lpnu) [inline ai_12/yana_vykliuk/epic_7_yana_vyklyuk/kr.cpp:40]: Task 2 (a,b) is not implemented according to requirements.
The overloading is not used

**Review by fm2lpnu — APPROVED** [review]

**Pull request merged**